packages/odyssey-react/src/components/Field/Field.tsx

```diff
@@ -175,10 +181,14 @@ function Hint({ id, children }: HintProps) {
 
 function FieldError({ id, children }: FieldErrorProps) {
   return (
-    <Box as="p" className={styles.error} id={`${id}-error`}>
-      <Text color="danger" fontSize="caption">
-        {children}
-      </Text>
+    <Box
```

Comment:
Given that we are making nearly everything theme-ready, I'll warn against using `Box` properties internally. It's going to be an odd experience when some Components have a component-theme and some rely on global tokens.

Ideally, we'd have a clear rule here, but our approach seems mixed in general.

Reply:
The mixed approach really threw me for a loop on this PR. 

Box has been removed.